src/pages/Home/Home.js

```diff
@@ -0,0 +1,90 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+import {Link, NavLink} from "react-router-dom";
+import Redditlogo from "../../assets/logo.png"
+
+function Home(props) {
+const [endpoint,setEndpoint] = useState(`https://www.reddit.com/hot.json?limit=15`)
+const [posts,setPosts]=useState([])
```

Comment:
Als `posts` uiteindelijk een object wordt, dan zou ik de default waarde aanpassen naar `undefined`. Dan heb je een duidelijker verschil tussen of het al wél of niet is ingeladen en wordt je check hier onderin ook wat simpeler
```suggestion
const [posts, setPosts]=useState();
```